plugins/operators/__init__.py

```
@@ -430,51 +399,20 @@ def resolve_input(self, ctx):
         field_name = ctx.params["field_name"]
         new_field_name = ctx.params["new_field_name"]
-        target = ctx.params.get("target", None)
-
-        target_view = _get_target_view(ctx, target)
+        target_view = ctx.target_view(param_name="target", require_flat=True)
 
         target_view.clone_sample_field(field_name, new_field_name)
         ctx.trigger("reload_dataset")
 
 
 def _clone_sample_field_inputs(ctx, inputs):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_clone_sample_field_inputs`

Add return type annotation: `None`

(ANN202)